docs/whats-new/preview-4.md

```diff
@@ -0,0 +1,608 @@
+
+Docker or Podman will be auto-detected; if both are present, Docker is preferred. Podman can be explicitly enabled/forced via an environment variable `DOTNET_ASPIRE_CONTAINER_RUNTIME=podman`.
+
+## Dashboard
+
+### Dashboard face lift
+
+:::image type="content" source="media/preview-4/dashboard-face-lift.png" lightbox="media/preview-4/dashboard-face-lift.png" alt-text=".NET Aspire Dashboard: Updates showing landing page.":::
+
+The dashboard has been updated with a new look and feel. The new dashboard is designed to reduce the space used by the navigation tabs and to make it easier to navigate between logs, metrics, and traces.
+
+### Running the Aspire dashboard standalone
+
+The Aspire dashboard can now be run as a standalone container image. This makes it easier to use the dashboard to manage applications that are running on a different machine or in a different environment. The dashboard can be used as an [OTLP](https://opentelemetry.io/docs/specs/otlp/) collector and viewer for applications that want to send and visualize telemetry data.
+
+Here's the command you can run to start the dashboard:
+
+```bash
+docker run --rm -it -p 18888:18888 -p 4317:18889 -d --name aspire-dashboard mcr.microsoft.com/dotnet/nightly/aspire-dashboard:8.0.0-preview.4
+```
+
+:::image type="content" source="media/preview-4/dashboard-as-container.png" lightbox="media/preview-4/dashboard-as-container.png" alt-text=".NET Aspire Dashboard: running in Docker Desktop.":::
+
+There are 2 ports that are exposed:
+
+1. The port that serves the dashboard UI (18888)
+2. The port that serves the OTLP grpc endpoint (18889)
+
+That will bring up a dashboard that you can use view logs, metrics, and traces from your applications. Here's a sample application that sends telemetry data to the dashboard.
+
+TODO: Sample pointing to the dashboard.  (@drewnoakes @JamesNK @kvenkatrajan)
+
+### Dashboard shortcuts
+
+The Aspire Dashboard now supports keyboard navigation via keyboard shortcuts. Click <kbd>Shift</kbd>+<kbd>?</kbd> to display the list of available shortcuts:
+
+:::image type="content" source="media/preview-4/dashboard-shortcuts.png" lightbox="media/preview-4/dashboard-shortcuts.png" alt-text=".NET Aspire Dashboard shortcuts dialog.":::
+
+### Metrics table view
+
+With preview 4, we have introduced a screen reader compatible table view for display of metrics data. This has similar options, as the graph, for filters and selecting the duration of time range for metrics display. The default data display is set to "Only show value updates" and can be toggled to display all data points.
+
+![image](https://github.com/dotnet/docs-aspire/assets/102772054/4622cf3a-dcf2-4149-9636-d1bef8184c5c)
+
+## Entity Framework and Aspire
+
+TODO: Changes to EF based components (@sebastienros @eerhardt)
+
```

Comment:
## Changes to database servers resources

With Preview 4 the `AddDatabase(string name)` method available on Database Servers resources has been improved such that the name of the resource (and as a consequence the connection name) that was registered can be different than the database name: `AddDatabase(string name, string databaseName = null)`

Here is an example that defines a database named `customers` and registers it as `crm`:

In your app host:

```csharp
builder.AddPostgres("postgres").AddDatabase("crm", "customers");
```

And in your application, resolve the component using the `crm` connection name:

```csharp
builder.AddNpgsqlDbContext<CustomerDbContext>("crm");
```

Reply:
I integrated these changes, thanks @sebastienros !